src/main/java/io/gatling/plugin/EnterprisePluginClient.java

```diff
@@ -33,12 +33,22 @@ public EnterprisePluginClient(EnterpriseClient enterpriseClient) {
   }
 
   @Override
-  public long uploadPackage(UUID packageId, File file) throws EnterpriseClientException {
-    nonNullParam(packageId, "packageId");
+  public long uploadPackage(UUID packageId, File file)
+      throws EnterpriseClientException {
     nonNullParam(file, "file");
+    nonNullParam(file, "packageId");
```

Comment:
non-blocking: same order as parameters
```suggestion
    nonNullParam(file, "packageId");
    nonNullParam(file, "file");
```